charts/vector/templates/NOTES.txt

```diff
@@ -4,23 +4,34 @@ __   __  __
   \_/  \/
 V E C T O R
 
```

Comment:
Still need to refactor this out into some `include` or `template` but I caught some UX issues in the meantime